benchmarks/examples/sombrero/sombrero.py

```diff
@@ -101,7 +104,7 @@ class SombreroBenchmark(SpackTest):
         },
         '*': {
             'flops': (1, None, None, 'Gflops/seconds'),
-        },
+            },
```

Comment:
I think this line was correctly indented before?
```suggestion
        },
```